### DISCUSSION TITLE
Change border color of TabViewItem

### DISCUSSION THREAD

**Discussion by cricketthomas** [amwx/FluentAvalonia#494] [Q&A] [answered]
Visual studio 2022 preview was released with a new design and I was trying to mimic the styles used on their tabs. 
However, I cannot figure how to restyle the curved edges paths properly.

FA Tabs on sample app with attempted overrides:
![image](https://github.com/amwx/FluentAvalonia/assets/15821271/2c7ccaca-5621-464b-b4c5-62cc0a74bcdb)
VS tabs:
![image](https://github.com/amwx/FluentAvalonia/assets/15821271/0eefa8a9-ea80-429c-b9e8-2c0f03092a2d)

    
I figured, just updating the values of the border colors should do the trick, however, after trying a lot of other ways, I cannot figure out how to get the paths and linear gradients to work as they currently do, just with different colors. 

Resources.xaml override:
```
<SolidColorBrush x:Key="TabViewSelectedItemBorderBrush" Color="Blue" />
<SolidColorBrush x:Key="TabViewBorderBrush" Color="Blue" />
```

Styles on user control in the sample app:
```
<UserControl.Styles>
  <Styles>
    <Style Selector="ui|TabView/template/ ContentPresenter">
        <Setter Property="BorderBrush" Value="Blue" />
    </Style>

    <Style Selector="ui|TabViewItem /template/ Path">
        <Setter Property="Fill" Value="Blue" />
    </Style>
    </Style>
</UserControl.Styles>
```

**Comment by amwx** [comment, marked as answer]
I don't think TabView supports this, at least the way its built. I see two options
1- Retemplate the control and define the geometry for this. This is the hardest option
2- Use a LinearGradientBrush to hide the bottom of the Border, though this only works if the height of the tab items don't change, 
```xaml
<LinearGradientBrush x:Key="TabViewSelectedItemBorderBrush"
                     StartPoint="0,0" EndPoint="0%,100%">
    <GradientStop Color="Blue" Offset="0" />
    <GradientStop Color="Blue" Offset=".9" />
    <GradientStop Color="Transparent" Offset="0.9" />
</LinearGradientBrush>
```
![image](https://github.com/amwx/FluentAvalonia/assets/40413319/59560397-2ca3-473b-afcb-f4d62aae8bd4)

**Reply by cricketthomas** [reply]
Thank you, this works!
